test/html-to-react-tests.js

```diff
@@ -403,6 +403,14 @@ describe('Html2React', function () {
 
         assert.equal(reactComponent.props.children.length, 5);
       });
+
+      it('Inline style should not contain usless spaces', function () {
```

Comment:
Please put this within the describe block 'parse valid HTML' and call it 'should handle spaces in inline styles'.